docs/index.md

```diff
@@ -75,12 +75,6 @@ Before submitting a new GitHub issue, please make sure to search for [existing G
 
 If that doesn't help, please [submit an issue](https://github.com/fastlane/fastlane/issues) on GitHub and provide information about your setup, in particular the output of the `fastlane env` command.
 
-## System requirements
-
-Currently, _fastlane_ is officially supported to run on macOS. 
-
-But we are working on 🐧 Linux and 🖥️ Windows support for parts of _fastlane_. Some underlying software like Xcode or iTunes Transporter is only available for macOS, but many other tools and actions can theoretically also work on other platforms. Please see [this Github issue for further information](https://github.com/fastlane/fastlane/issues/11687).
-
```

Comment:
```md
## System requirements

_fastlane_ is officially supported to run on macOS.

🐧 Linux and 🖥️ Windows are partially supported. Some underlying software like Xcode are only available for macOS but many other tools, actions, and the `spaceship` module can work on other platforms.
```

Reply:
I agree! This is exactly where _fastlane_ is at as of today. The wording is just confusing as it is currently, and your suggestion makes it 100% clear 💯

Reply:
@joshdholtz I applied your changes :) think this is g2g?

Reply:
Yes yes! Thanks for applying the change ❤️